modules/oe_starter_content_documents_field/README.md

```diff
@@ -0,0 +1,3 @@
+# OpenEuropa Starter Content Documents Field
+
+This module provides the Documents (oe_documents) Field storage field, by enabling it you will be able to add Documents field to your entities. Documents fields will be automatically styled when using OpenEuropa Theme.
```

Comment:
listing modules which are automatically enabled (because they are set as dependencies) helps no one. If you list requirements list what needs to be added in composer.json (if anything but double check)